tests/unit/utils/LocationStatusStream.spec.js

```diff
@@ -30,21 +30,12 @@ describe('LocationStatusStream', () => {
                 collection: collectionStub,
                 command: mongoCommandStub,
             });
-            const mongoConnectStub = sinon.stub(MongoClient, 'connect').resolves({
+            sinon.stub(MongoClient.prototype, 'connect').resolves({
                 db: dbStub,
             });
             const lss = new LocationStatusStream('lifecycle', mongoConfig, null, null, FakeLogger);
             lss._setupMongoClient(err => {
                 assert.ifError(err);
```

Comment:
we should still check that it was called with the expected url, i.e
* at least `assert(mongoConnectStub.calledOnce)`
* to partially check params, with `calledOnceWith` is not practical, we may simply assert on (a subset of) the params of the call afterwarsd, something like `assert(mongoConnectStub.call(0).args[1] == mongoUrl)`